introduction.md

```diff
@@ -1 +1,5 @@
-I love Ruby and JavaScript!
\ No newline at end of file
+I love Ruby and JavaScript!
+Skills:
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?